index.html

```diff
@@ -2645,6 +2657,164 @@ <h3>Refreshing</h3>
+                <th>Description</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>id</td>
+                <td>
+A URL that dereferences to an SVG image [[SVG]] with an associated
+`image/svg+xml` media type.
+                </td>
+              </tr>
+              <tr>
+                <td>type</td>
+                <td>
+The <code>type</code> property MUST be <code>SvgRenderingTemplate2023</code>.
+                </td>
+              </tr>
+              <tr>
+                <td>digestMultibase</td>
+                <td>
+An optional multibase-encoded multihash of the SVG image. The multibase value
+MUST be `z` and the multihash value MUST be SHA-2 with 256-bits of output
+(`0x12`).
+<p class="issue">
+The Working Group is seeking feedback related to the mechanism used to express
+the cryptographic hash of the SVG image. This property is a placeholder for that
+discussion and is not meant to imply a final decision related to the way this
+will be done if the feature is accepted in the future.
+</p>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+
+          <p>
+The data model shown above is expressed in a <a>verifiable credential</a>
+in the example below.
+          </p>
+
+          <pre class="example nohighlight"
+          title="Usage of the render property by an issuer">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "issuer": "https://example.edu/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  <span class="highlight">"render": [{
+    "id": "https://example.edu/credentials/BachelorDegree.svg",
+    "type": "SvgRenderingTemplate2023",
+    "digestMultibase": "zQmAPdhyxzznFCwYxAp2dRerWC85Wg6wFl9G270iEu5h6JqW"
+  }]
+  </span>
+}
+        </pre>
+
+        <p>
+In the example above, the <a>issuer</a> has provided an SVG rendering template
+for a bachelor's degree that will be filled in with the information in the
+<a>verifiable credential</a>.
+        </p>
+
+        <section>
+          <h5>SvgRenderingTemplate2023 Algorithm</h5>
+
+          <p>
+The following algorithm is used to transform the SVG image template into the
+final SVG image that is displayed. The inputs to the algorithm are the
+<a>verifiable credential</a> (`verifiableCredential`) and the SVG image
+source code (`svgImage`). The output is a SVG image.
+          </p>
+
+          <ol class="algorithm">
+            <li>
+Generate a map, `replacementMap`, by finding all strings in `svgImage` that
+start with `{{` (double open braces) and end with `}}`
```

Comment:
Instead of defining a one-off interpolation algorithm, perhaps it would make sense to define this algorithm in terms of svg/xml. e.g. replace `<vc-select selection="{jsonpointer}" />`

I think this would have the nice property of making the SVGRenderingTemplate2023 still be valid svg? svg2 says unknown elements should render as `<g />`

(`{{syntax}}` may also be valid svg, but I imagine they'd render ugly)

Reply:
it might also scale to more complex types of replacements later (e.g. 'selections' with more complex types than just a `jsonpointer`)